src/workerd/api/sql.c++

```diff
@@ -273,6 +379,116 @@ jsg::Ref<SqlStorage::Cursor> SqlStorage::Statement::run(jsg::Arguments<BindingVa
   return result;
 }
 
+jsg::Ref<SqlStorage::BoundStatement> SqlStorage::Statement::bind(
+    jsg::Arguments<BindingValue> bindings) {
+  return jsg::alloc<BoundStatement>(JSG_THIS, kj::mv(bindings));
+}
+
+jsg::JsArray SqlStorage::Statement::raw(
+    const v8::FunctionCallbackInfo<v8::Value>& args,
+    const jsg::TypeHandler<BindingValue>& bindingTypeHandler,
+    const jsg::TypeHandler<RawOptions>& optionsTypeHandler) {
+  jsg::Lock& js = jsg::Lock::from(args.GetIsolate());
```

Comment:
Nit: does this need to use `args.GetIsolate()` or could the `jsg::Lock&` be passed in?

Reply:
At present, the JSG stuff for wrapping methods does not support taking `jsg::Lock&` and `const v8::FunctionCallbackInfo<v8::Value>&` at the same time. It should probably be added but I didn't really want to get into JSG improvements in this PR...